README.md

```diff
@@ -103,7 +103,7 @@ official compatibility chart below.
 
 *Environment variables:*
 
-- `$KUBECONFIG`: points to one or multiple valid kubeconfig files that include the context you want to deploy to. File names are separated by colon for Linux and Mac, and semi-colon for Windows. If ommitted, will use the Kubernetes default of `~/.kube/config`.
+- `$KUBECONFIG`: points to one or multiple valid kubeconfig files that include the context you want to deploy to. File names are separated by colon for Linux and Mac, and semi-colon for Windows. If omitted, will use the Kubernetes default of `~/.kube/config`.
```

Comment:
While we're here...

>If omitted, will use the Kubernetes default....

Change to:
_If omitted, Krane will use the Kubernetes default..._

Reply:
Changed as requested 😃